go/vt/vtgate/planbuilder/testdata/aggr_cases.json

```diff
@@ -824,6 +824,66 @@
       ]
     }
   },
+  {
+    "comment": "group concat with a separator needing evaluation on vtgate",
+    "query": "select group_concat(music.name SEPARATOR ', ') as `Group Name` from user join user_extra on user.id = user_extra.user_id left join music on user.id = music.id group by user.id;",
+    "plan": {
+      "QueryType": "SELECT",
+      "Original": "select group_concat(music.name SEPARATOR ', ') as `Group Name` from user join user_extra on user.id = user_extra.user_id left join music on user.id = music.id group by user.id;",
+      "Instructions": {
+        "OperatorType": "Aggregate",
+        "Variant": "Ordered",
+        "Aggregates": "group_concat(0) AS Group Name",
```

Comment:
should separator be part of plan description?

Reply:
We saw that, but right now we're just using the `AggrName` for printing. We'd have to print the original expr or have a switch case specificaly for GroupConcat.